webcomponents/tool/src/main/frontend/js/sakai-permissions.js

```diff
@@ -51,6 +51,8 @@ class SakaiPermissions extends SakaiElement {
 
     return {
       tool: String,
+      groupReference: { attribute: 'group-reference', type: String }, //remember to set this attribute before tool, or the loaded permissions will be wrong
+      disableGroups: { attribute: 'disabled-groups', type: Boolean }, //remember to set this attribute before tool, or the loaded permissions will be wrong
```

Comment:
You should either document this at the top of the class, using jsdoc syntax, or you could control the state by implementing setters for groupReference and disableGroups. I understand this issue with attribute order and I've wrestled with it myself. I usually sort it out by using setters. But, just documenting this in the class js doc will do, with a tag example

Reply:
Done